docs/sources/panels-visualizations/visualizations/state-timeline/index.md

```diff
@@ -33,7 +33,7 @@ The state timeline visualization is useful when you need to monitor and analyze
 
 ## Configure a state timeline
 
-Once you have [created a dashboard](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/dashboards/build-dashboards/create-dashboard/), the following video shows you how to configure a state timeline:
+Once you have [created a dashboard](/docs/grafana/<GRAFANA_VERSION>/dashboards/build-dashboards/create-dashboard/), the following video shows you how to configure a state timeline:
```

Comment:
We've made a lot of changes to our linking styles in the past few months, so they're hard to keep track of, but we prefer full URLs in this case.
```suggestion
```